GTBitsOfGood/canine-assistants
# Individual Dog Access Level Views

## Individual Dog Access Level Views

Issue Number(s): #134 

What does this PR change and why?
Conditionally renders frontend depending on the user role/association with the dog. Also implements backend logic for returning correct forms and logs depending on user and dog.

### Checklist


### Critical Changes

- Database change / migration to run
- Environment config change
- Breaking API change

### Related PRs

PR Number(s): 

### How To Test

Switch through admin/manager/volunteer/instructor roles to see views change depending on association to dog
